core/cmd/shell.go

```diff
@@ -381,7 +381,7 @@ func handleNodeVersioning(db *sqlx.DB, appLggr logger.Logger, rootDir string, cf
 	return nil
 }
 
-func takeBackupIfVersionUpgrade(dbUrl url.URL, rootDir string, cfg periodicbackup.BackupConfig, lggr logger.Logger, appv, dbv *semver.Version) (err error) {
+func takeBackupIfVersionUpgrade(dbUrl url.URL, rootDir string, cfg periodicbackup.BackupConfig, lggr logger.Logger, appv, dbv *semver.Version, healhReportPort uint16) (err error) {
```

Comment:
healhReportPort -> healthReportPort

Reply:
Fixed in [`eeeba73` (#9895)](https://github.com/smartcontractkit/chainlink/pull/9895/commits/eeeba73b0028924f32c2ae26a4f524be4e5c9068)